python/social_bot/__init__.py

```diff
@@ -57,6 +63,12 @@
     max_episode_steps=200,
 )
 
+register(
+    id='SocialBot-GroceryGroundImageLanguage-v1',
+    entry_point='social_bot.envs:GroceryGroundImageLanguage',
+    max_episode_steps=1e8,
```

Comment:
There is no need to register another one if the difference with v0 is only max_episode_steps, max_episode_steps can be configured by suite_gym.load.max_episode_steps=1000

Reply:
Good to know!